src/types/excelreader/python.rs

```diff
@@ -83,12 +83,15 @@ impl ExcelReader {
             .into_pyresult()?
             .to_owned();
 
+        let range = py
+            .allow_threads(|| {
+                self.sheets
+                    .with_header_row(calamine_header_row)
+                    .worksheet_range(&sheet_meta.name)
+            })
+            .into_pyresult()?;
```

Comment:
Please leave every implem in its branch: the first one uses `worsheet_range_ref`, which reduces fastexcel's memory usage by working on borrowed types rather than owned ones